extra/smtp-testing/conductor-workers-smtp-test.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+
+    mender-conductor-workers:
+        environment:
+            - CONDUCTOR=http://mender-conductor:8080
+            - SMTP_ADDRESS=172.29.0.107:4444
```

Comment:
This is the address of the host (on which the tests are run) and needs to be passed from the test